RecoMET/METAlgorithms/src/METSignificance.cc

```diff
@@ -103,7 +97,10 @@ metsig::METSignificance::getCovariance(const edm::View<reco::Jet>& jets,
 
        //dP4 recovery
        for( const auto& it : footprint) {
-	 if( (it->p4()-(*pfCandidates)[i].p4()).Et2()<0.000025 ){
+	 if((it.isNonnull()) && (it.isAvailable()) &&
+	    (((!useDeltaRforFootprint_) && ((it->p4()-(*pfCandidates)[i].p4()).Et2()<0.000025)) ||
+	    (( useDeltaRforFootprint_) && (reco::deltaR2(it->p4(),(*pfCandidates)[i].p4())<0.00000025)))){
```

Comment:
```suggestion
	    (( useDeltaRforFootprint_) && (reco::deltaR2(*it,(*pfCandidates)[i])<0.00000025)))){
```
you may want to pick up this optimization from #29919